downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md

```diff
@@ -22,9 +24,255 @@ However, you need to use the following commands instead of the above if you have
 
 If you have not installed jBallerina, then download the [installers](https://ballerina.io/downloads/) to install.
 
+### Highlights
+
+- Support for extern methods in objects
+- Support for executing stored procedures in the SQL connector
+- Azure Functions support
+
+### What's new in Ballerina Swan Lake Preview 3
+
+#### Language
+
+The language implementation is based on [Ballerina Language Specifications Draft 2020-06-18](https://ballerina.io/spec/lang/draft/v2020-06-18/).
+
+##### Extern method support
```

Comment:
Use the latest content in the doc for this section

Reply:
Done.